projects/control-service/projects/pipelines_control_service/src/main/java/com/vmware/taurus/service/repository/JobsRepository.java

```diff
@@ -33,6 +34,12 @@
  */
 @Repository
 public interface JobsRepository extends PagingAndSortingRepository<DataJob, String> {
+
+  @NotNull
+  @Override
+  @Query("select j from DataJob j left join fetch j.dataJobDeployment")
```

Comment:
I assume this is the optimization? Explain the the PR description (which would become also commit description) what was the problem before and why this works better.

Reply:
Correct, I will update it.